telco-core/configuration/reference-crs-kube-compare/required/networking/metallb/bfd-profile.yaml

```diff
@@ -12,7 +12,9 @@ spec:
   echoInterval: {{ if .spec.echoInterval }} {{ .spec.echoInterval }} {{ else }} "50ms" {{ end }} # default 50ms
   {{ end }}
   detectMultiplier: {{ if .spec.detectMultiplier }} {{ .spec.detectMultiplier }} {{ else }} "3" {{ end }} # default 3
-  echoMode: true
+  {{ - if .spec.echoMode }}
```

Comment:
```suggestion
  {{- if .spec.echoMode }}
```
The `-` must be contiguous with the `{{`

Reply:
updated